packages/autofocus/src/index.ts

```diff
@@ -0,0 +1,62 @@
+import { createEffect, onMount, JSX, Accessor } from "solid-js";
+import { FalsyValue } from "../../utils";
```

Comment:
utils package have to be imported and added to package.json file
```suggestion
import { FalsyValue } from "@solid-primitives/utils";
```